pkg/cli/sql_test.go

```diff
@@ -113,21 +137,28 @@ func Example_sql() {
 	// > select 123 as "123"
 	// 123
 	// 123
+	// sql -e select 123 --set display_format=raw
+	// # 1 column
+	// # row 1
+	// ## 3
+	// 123
+	// # 1 row
+	// sql -e \set display_format=raw -e select 123 as "123"
+	// # 1 column
+	// # row 1
+	// ## 3
+	// 123
+	// # 1 row
 	// sql --set unknownoption -e select 123 as "123"
 	// invalid syntax: \set unknownoption. Try \? for help.
 	// ERROR: invalid syntax
 	// sql --set display_format=invalidvalue -e select 123 as "123"
 	// \set display_format invalidvalue: invalid table display format: invalidvalue (possible values: tsv, csv, table, records, sql, html, raw)
 	// ERROR: invalid table display format: invalidvalue (possible values: tsv, csv, table, records, sql, html, raw)
-	// sql -e select 1/(@1-2) from generate_series(1,3)
```

Comment:
was deleting these lines intended?

Reply:
They are still there! Moved up to the previous test see line 103